tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:

```

Comment:
Shouldn't need the try except anymore - unittest.mock has been in Python since 3.3

Reply:
good to know, will fix it